modules/onboard-aws-rds-aurora-mysql-kinesis/variables.tf

```diff
@@ -92,44 +76,66 @@ variable "cluster_db_subnet_group_name" {
   default     = null
 }
 
-variable "cluster_vpc_security_group_ids" {
-  description = "List of VPC security groups to associate."
+variable "cluster_enabled_cloudwatch_logs_exports" {
+  description = "Set of log types to enable for exporting to CloudWatch logs. Valid values: audit, error, general, slowquery."
   type        = list(any)
   default     = null
 }
 
-variable "cluster_maintenance_schedule" {
-  description = "Weekly time range during which system maintenance can occur, in (UTC)."
+variable "cluster_engine_version" {
+  description = "Database engine version, i.e. 8.0.mysql_aurora.3.05.1"
   type        = string
   default     = null
 }
 
-variable "cluster_final_snapshot" {
-  description = "Determines whether a final DB snapshot is created before the DB cluster is deleted. If true is specified, no DB snapshot is created. If false is specified, a DB snapshot is created before the DB cluster is deleted, using the value from final_snapshot_identifier. Default is false"
-  type        = bool
+variable "cluster_master_password" {
+  description = "Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file. Cannot be set if manage_master_user_password is set to true"
+  type        = string
+}
+
+variable "cluster_master_username" {
+  description = "Username for the master DB user, must not use rdsadmin as that is reserved."
+  type        = string
+  validation {
+    condition = (
+      var.cluster_master_username != "rdsadmin"
+    )
+    error_message = "The aurora cluster master username must not be either \"rdsadmin\". Please select another value."
+  }
+}
+
+variable "cluster_network_type" {
+  description = " Network type of the cluster. Valid values: IPV4, DUAL"
+  type        = string
   default     = null
 }
 
-variable "cluster_db_port" {
+variable "cluster_port" {
   description = "Port on which the DB accepts connections."
   type        = number
   default     = null
 }
 
-variable "cluster_apply_immediately" {
-  description = "Specifies whether any cluster modifications are applied immediately, or during the next maintenance window. Default is true"
-  type        = bool
+variable "cluster_preferred_maintenance_window" {
+  description = "Weekly time range during which system maintenance can occur, in (UTC)."
+  type        = string
   default     = null
 }
 
-variable "cluster_parameter_group_name" {
-  description = "Cluster parameter group associated with the cluster"
-  type        = string
+variable "cluster_skip_final_snapshot" {
+  description = "Determines whether a final DB snapshot is created before the DB cluster is deleted. If true is specified, no DB snapshot is created. If false is specified, a DB snapshot is created before the DB cluster is deleted, using the value from final_snapshot_identifier. Default is false"
```

Comment:
"default is false" - I'm just wrong here.

Good opportunity to clean the other variable descriptions that say "default is x" and don't match what our actual default is set to.